api/services/mailer/index.js

```diff
@@ -30,7 +32,33 @@ async function sendUAAInvite(email, link) {
   });
 }
 
+async function sendSandboxReminder(organization) {
+  const dateStr = moment(organization.sandboxCleaningScheduledAt).format('MM-DD-YYYY');
+
+  const {
+    id: organizationId,
+    name: organizationName,
+    Users,
+  } = organization;
+
+  const organizationLink = `${hostname}/organizations/${organizationId}`;
+  const subject = `Your Pages sandbox organization's sites will be deleted in ${organization.daysUntilSandboxCleaning} days`;
+
+  ensureInit();
+
+  return mailQueue.add('sandbox-reminder', {
+    to: Users.map(user => user.email).join('; '),
```

Comment:
@amirbey Did you have a chance to look at this one? The Github UI may have hidden it...

Reply:
@davemcorwin 

1. updated recipients to be an array
2. since the users are managers in the same org self-administered not sure they need to be bcc? 🤔

Reply:
Thank about when you get an automated email from a system, it's always an individual email. It's not sent to a group of people, everyone gets their own. Sending to a group also may lead to replies as the group may try to coordinate and we we don't want that, not that we receive replies.